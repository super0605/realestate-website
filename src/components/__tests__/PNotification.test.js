import * as React from "react";
import { mount, shallow } from "enzyme";
import pNotification from "../pNotification";

describe("pNotification Component", () => {
  let props;
  let mountedpNotification;
  let wrapper;

  const renderWrapper = (bShallow) => {
    if (!mountedpNotification) {
      mountedpNotification = bShallow
        ? shallow(<pNotification {...props} />)
        : mount(<pNotification {...props} />);
    }
    return mountedpNotification;
  };

  beforeEach(() => {
    wrapper = renderWrapper(true);
  });

  it("should render without problems", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
