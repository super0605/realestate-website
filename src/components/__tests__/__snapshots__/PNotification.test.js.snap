// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pNotification Component should render without problems 1`] = `<pNotification />`;
